File: C++/Step11-Sorting/1427.cpp
//
//  1427.cpp
//  Step11-Sorting
//
//  Created by 김예빈 on 2019. 9. 9..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main(int argc, const char * argv[]) {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int n;
    
    cin >> n;
    
    vector<int> num;
    
    while(n > 0) {
        num.push_back(n % 10);
        n /= 10;
    }
    
    sort(num.begin(), num.end(), greater<int>());
    
    for(int i = 0; i < num.size(); i++)
        cout << num[i];
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step12-Sorting/1427.swift
//
//  1427.swift
//  Step11-Sorting
//
//  Created by Yebin Kim on 2019/12/2.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

if let N = readLine() {
    let arr = N.compactMap{ Int(String($0)) }.sorted(by: >).map{ String($0) }
    print(arr.reduce(""){ $0 + $1 })
}
